R/zzz.R

```diff
@@ -9,9 +9,22 @@
 #' @importFrom withr with_options
 "_PACKAGE"
 
+register_mlr3 = function() {
+  x = utils::getFromNamespace("mlr_reflections", ns = "mlr3")
+  x$pipeops$valid_tags = unique(c(x$pipeops$valid_tags,
+    c("abstract", "meta", "missings", "feature selection", "imbalanced data",
+    "data transform", "target transform", "ensemble", "robustify", "learner", "encode")))
+}
 
 .onLoad = function(libname, pkgname) {  # nocov start
-
+  register_mlr3()
+  setHook(packageEvent("mlr3", "onLoad"), function(...) register_mlr3(), action = "append")
   backports::import(pkgname)
-
 }  # nocov end
+
+.onUnload = function(libpath) { # nocov start
```

Comment:
what is this for?

Reply:
I think this makes sure that unloading the package cleanly removes the additions made by `register_mlr3`.
I copied this over from https://github.com/mlr-org/mlr3proba/blob/master/R/zzz.R

Reply:
I guess this is how this is done now...